libs/wingsdk/src/http/http.ts

```diff
@@ -0,0 +1,259 @@
+import { Code, InflightClient } from "../core";
+
+/**
+ * RequestCache-
+ * The cache read-only property that contains the cache mode of the request.
+ * It controls how the request will interact with the browser's HTTP cache.
+
+ */
+export enum RequestCache {
```

Comment:
The docs information here is great. One thing we should update is since we don't know whether the code will be running on a browser or on a cloud serverless function or elsewhere, let's avoid using the term "browser" and instead use "system" or "runtime environment" throughout the descriptions.